DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -189,5 +189,19 @@
   <class name="edm::Wrapper< MTDTrackingDetSetVector >"/>
   <class name="MTDTrackingOwnVector"/>
   <class name="edm::Wrapper< MTDTrackingOwnVector >"/>
+  <class name="VectorHit" ClassVersion="3">
+   <version ClassVersion="3" checksum="546506376"/>
+  </class>
+  <class name="VectorHit2D" ClassVersion="3">
+   <version ClassVersion="3" checksum="333065919"/>
+  </class>
```

Comment:
it looks like this is never persisted; a dictionary is not needed then

Reply:
Removed